extensions/adapters/vector/src/main/java/mil/nga/giat/geowave/adapter/vector/util/FeatureDataUtils.java

```diff
@@ -76,4 +82,56 @@ else if (crs != null) {
 		}
 		return defaultCRSEntry;
 	}
+
+	public static String getAxis(
+			final CoordinateReferenceSystem crs ) {
+		final CoordinateSystem cs = crs == null ? null : crs.getCoordinateSystem();
+		if (cs != null && cs.getDimension() > 0) return cs.getAxis(
+				0).getDirection().name().toString();
+		return "";
```

Comment:
do we really want to return an empty string if we can't get a CoordinateSystem?   
If we don't want to deal with error propagation back for a nonsense case them maybe use a guava precondition instead?